### PR TITLE
Browse all category made responsive

### PR DESCRIPTION
**Pull request by captainsahil001**
Made the browse all category button responsive and working #34 

## Description

Added all the subcategories of the button into a grid container and made it responsive and can be clicked to viewed using feature show more and show less

## Motivation and Context
It was one of the major bug and was needed to be fixed so made it alright now.

## Screenshots (if appropriate):
Screenshot 2024-06-04 213629
Screenshot 2024-06-04 213645

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by captainsahil001** [comment]
pls check i have completed my task of issue #34 and merge to make my contribution in this project

File: src/assets/images/pet.svg
Comment (kushagra-singhh) [inline src/assets/images/pet.svg:1]: The dimensions of this image is not at par with others.

**Review by kushagra-singhh — CHANGES_REQUESTED** [review]

File: src/assets/images/pet.svg
Comment (captainsahil001) [reply src/assets/images/pet.svg:1]: yes it was according to mavrick

Comment (MAVRICK-1) [reply src/assets/images/pet.svg:1]: @kushagra-singhh is right , @captainsahil001 can you update it

Comment (captainsahil001) [reply src/assets/images/pet.svg:1]: > @kushagra-singhh is right , @captainsahil001 can you update it

Can you suggest me the dimension which would be appropriate and which image you are talking about can you point it , are u talking abt the icon images.

Comment (kushagra-singhh) [reply src/assets/images/pet.svg:1]: pet.svg is smaller than the other images.

Comment (captainsahil001) [reply src/assets/images/pet.svg:1]: > pet.svg is smaller than the other images.

I think only this pet and milk one I need to set the dimension right

Comment (kushagra-singhh) [reply src/assets/images/pet.svg:1]: Yes.

**Review by kushagra-singhh — APPROVED** [review]

**Review by captainsahil001 — COMMENTED** [review]
made the changes kindly check

**Comment by kushagra-singhh** [comment]
> made the changes kindly check

Looks good !! 👍🏻

**Comment by MAVRICK-1** [comment]
@captainsahil001 great work @kushagra-singhh thanx 4 the help

**Review by captainsahil001 — COMMENTED** [review]
ok

**Pull request merged**